modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -108,12 +111,40 @@ export default {
           doc._fields = this.subfields[doc._id];
         }
       }
-    },
-    checked() {
-      this.updateCheckedDocs();
```

Comment:
This method was overriden in AposPagesManager, it's not needed anymore. It's also unused in our modules.
Wondering if it's considered a BC